Updated the service.

millerscout opened the pull request:
- updated example in folder.
- updated service, so that's possible to make chainable methods.
- removed promises... it's better to call the callback instead it's a straight forward solution :P
- removed deferList (not needed anymore)
- updated directive  so it's not listening to an trigger anymore.

updated version and build

Update README.md

millerscout commented:
I was working on the typescript version, while i was re-writing some parts i noticed some things could be simplified, so i did it :)

mendhak commented on example/app.js:
```diff
@@ -61,7 +38,71 @@ app.controller('MyController', function ($scope) {
+          showBullets: true,
+          exitOnOverlayClick: false,
+          exitOnEsc:false,
+          nextLabel: '<strong>Proximo!</strong>',
```
Please can you keep these in English

millerscout replied:
sure , i'll look onto that tonight

millerscout commented:
i reviewed the code the code again, and updated the examples and readme.md.

- added a small feature, that i miss on introJs, disable/enable buttons when the introjs is open when the button has the directive on it.
